templates/.reek.yml

```diff
@@ -13,6 +13,16 @@ directories:
     "app/helpers":
       UtilityFunction:
         enabled: false
+      FeatureEnvy:
+        enabled: false
+    "app/jobs":
+      UtilityFunction:
+        enabled: false
+      InstanceVariableAssumption:
```

Comment:
the `InstanceVariablesAssumption` is a really helpful check. I'd like to discuss the removal of this one.